signal/signal_windows.go

```diff
@@ -17,10 +19,18 @@ const (
 // SignalMap is a map of "supported" signals. As per the comment in GOLang's
 // ztypes_windows.go: "More invented values for signals". Windows doesn't
 // really support signals in any way, shape or form that Unix does.
-//
-// We have these so that docker kill can be used to gracefully (TERM) and
-// forcibly (KILL) terminate a container on Windows.
 var SignalMap = map[string]syscall.Signal{
-	"KILL": syscall.SIGKILL,
-	"TERM": syscall.SIGTERM,
+	"HUP":    syscall.Signal(windows.SIGHUP),
+	"INT":    syscall.Signal(windows.SIGINT),
+	"QUIT":   syscall.Signal(windows.SIGQUIT),
+	"SIGILL": syscall.Signal(windows.SIGILL),
```

Comment:
It is SIGSIGILL since 2017. Looks like a typo though...

https://github.com/containerd/containerd/pull/743/files

Reply:
Yeah I think it's likely a typo in the original.

Reply:
Especially since CI is passing now :laughing: